### ISSUE TITLE
Enhance Validations documentation

### ISSUE THREAD

**Issue by luislobo** [balderdashy/sails-docs#284]
In: 
https://github.com/balderdashy/sails-docs/blob/master/concepts/ORM/Validations.md

there are descriptions like: Where are params defined?, something to do with REGEX, Something about regexes

Also, samples on how to use "normal" validations is needed, besides the only sample below, about using custom validations.

**Comment by rachaelshaw** [comment]
@luislobo would you be down to take a stab at a PR?

**Comment by luislobo** [comment]
@rachaelshaw Yeah, sure, I could do some during the weekend

**Comment by rachaelshaw** [comment]
@luislobo sounds great. Thank you.

**Comment by sailsbot** [comment]
Thanks for posting, @luislobo.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
